src/jira-issues-provider.ts

```diff
@@ -59,17 +59,21 @@ export class JiraIssuesProvider implements TreeDataProvider<TreeItem> {
 				.then(async function(data: any){
 					var children: JiraIssue[] = [];
 					var promises = data.issues.map( async(issue: any) => {
-						const jiraIssue = new JiraIssue(`${issue.key}: ${issue.fields.summary}`, issue);
-						const issueType = issue.fields.issuetype.name;
-						var icon = `${issueType.toLowerCase()}.svg`;
+						var description = `${issue.key}: (${issue.fields.status.name}) ${issue.fields.summary}`
+						const jiraIssue = new JiraIssue(description, issue);
+						if (issue.fields.status.name == "Done") {
```

Comment:
The identifier you're using for an issue being "done" is specific to the project. On my project for example an issue that is done has a status of "Complete". We want something a bit more generic that will apply to all projects. I looked around a bit and I think `issue.fields.resolution != null` might be what we're looking for.

Reply:
Sounds good. I'll update that.